.vib/drupal/goss/vars.yaml

```diff
@@ -0,0 +1,31 @@
+binaries:
+  - apachectl
+  - gosu
+  - mysql
+  - php
+  - render-template
+  - drush
+files:
+  - paths:
+    - /opt/bitnami/drupal/sites/default/settings.php
+    - /opt/bitnami/drupal/sites/default/files/.htaccess
```

Comment:
There is no longer need to have these files here:

- There is a more advanced tests for `/opt/bitnami/drupal/sites/default/settings.php` already in `drupal.yaml`
- You are checking `/opt/bitnami/drupal/sites/default/settings.php`, which should result from `/opt/bitnami/drupal/sites/default/files/.htaccess`. If the latter does not exist, the changes should neither be present in the former 😁